ROADMAP.md

```diff
@@ -0,0 +1,70 @@
+# Roadmap
```

Comment:
It's weird to have two Readmap headers?

Reply:
Oh, I see. Can we move this into that `ROADMAP.md` for now?

Reply:
Its in at file already as far as I can tell.